pandas/tests/indexes/datetimes/test_astype.py

```diff
@@ -301,6 +301,19 @@ def test_to_period_nofreq(self):
         assert idx.freqstr is None
         tm.assert_index_equal(idx.to_period(), expected)
 
+    @pytest.mark.parametrize('tz', [None, 'US/Central'])
+    def test_astype_category(self, tz):
+        obj = pd.date_range("2000", periods=2, tz=tz)
+        result = obj.astype('category')
+        expected = pd.CategoricalIndex([pd.Timestamp('2000-01-01', tz=tz),
+                                        pd.Timestamp('2000-01-02', tz=tz)])
+        tm.assert_index_equal(result, expected)
+
```

Comment:
passing copy= flag?

Reply:
pls do this

Reply:
can you clarify what you're asking for here?

Reply:
i guess since we don't have a copy flag yet (did you create an issue), can't check if this is actually a copy yet